src/libraries/System.Text.Json/tests/System.Text.Json.FSharp.Tests/EnumTests.fs

```diff
@@ -0,0 +1,51 @@
+﻿module System.Text.Json.Tests.FSharp.EnumTests
+
+open System
+open System.Text.Json
+open System.Text.Json.Serialization
+open Xunit
+
+[<Flags>]
+type BadEnum =
+  | ``There's a comma, in my name`` = 1
+  | ``There's a comma, even here`` = 2
+  | ``ThisisagoodEnumValue`` = 4
+
+let badEnum = BadEnum.``There's a comma, in my name`` ||| BadEnum.``There's a comma, even here``
+let badEnumJsonStr = @$"""{badEnum.ToString()}"""
```

Comment:
```suggestion
let badEnumJsonStr = badEnum.ToString()
```

Reply:
The json value of enum should has double quotes, right?